include/fmt/ranges.h

```diff
@@ -155,7 +155,7 @@ template <typename T>
 struct has_mutable_begin_end<
     T, void_t<decltype(detail::range_begin(std::declval<T>())),
               decltype(detail::range_end(std::declval<T>())),
-              enable_if_t<std::is_copy_constructible<T>::value>>>
+              int>>
```

Comment:
Please add a comment briefly explaining why we have int here.

Reply:
Sure.